src/content/docs/es/guides/cms/decap-cms.mdx

```diff
@@ -8,8 +8,125 @@ i18nReady: true
+         fields: # Los campos para cada documento, generalmente al principio
+            - { label: "Layout", name: "layout", widget: "hidden", default: "blog" }
+            - { label: "Title", name: "title", widget: "string" }
+            - { label: "Publish Date", name: "date", widget: "datetime" }
+            - { label: "Featured Image", name: "thumbnail", widget: "image" }
+            - { label: "Rating (scale of 1-5)", name: "rating", widget: "number" }
+            - { label: "Body", name: "body", widget: "markdown" }
+     ```
+
+4. Agrega la ruta `admin` para tu aplicación React. Este archivo puede ser `public/admin/index.html` junto con `config.yml` o, si prefieres utilizar una ruta Astro, `src/pages/admin.astro`.
+
+     <FileTree>
+     - public
+       - admin
+          -config.yml
+          - index.html
+     </FileTree>
+     ```html title="/public/admin/index.html" {7, 11}
+     <!doctype html>
+     <html lang="es">
+       <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta name="robots" content="noindex" />
+        <link href="/admin/config.yml" type="text/yaml" rel="cms-config-url" />
+        <title>Administrador de contenidos</title>
+       </head>
+      <body>
+        <script src="https://unpkg.com/decap-cms@^3.1.2/dist/decap-cms.js"></script>
+      </body>
+    </html>
+    ```
+
+5. Para habilitar la carga de medios a una carpeta específica a través del editor Decap, agregue una ruta adecuada:
+     ```yml title="/public/admin/config.yml"
+     media_folder: "src/assets/images" # Ubicación donde se almacenarán los archivos en el repositorio
+     public_folder: "src/assets/images" # El atributo src para los medios cargados
+     ```
+
+Consulta la [documentación de configuración de Decap CMS](https://decapcms.org/docs/configure-decap-cms/) para obtener instrucciones y opciones completas.
+
+## Uso
+Navega hasta `yoursite.com/admin/` para usar el editor Decap CMS.
+
+## Autenticación
+
+### Decap CMS con Netlify Identity
+
+Decap CMS fue desarrollado originalmente por Netlify y tiene soporte de primera clase para [Netlify Identity](https://docs.netlify.com/security/secure-access-to-sites/identity/).
+
+Al implementar en netlify, configura la identidad para tu proyecto a través del panel de Netlify e incluye el [Netlify Identity Widget](https://github.com/netlify/netlify-identity-widget) en la ruta `admin` de tu proyecto. Opcionalmente, incluye el widget de identidad en la página de inicio de tu sitio si planeas invitar a nuevos usuarios por correo electrónico.
+
+### Decap CMS con clientes OAuth externos
+
+Al implementar en proveedores de hosting distintos de Netlify, debes crear tus propias rutas OAuth.
+
+En Astro, esto se puede hacer con rutas renderizadas bajo demanda en tu proyecto configurado con [salida `servidor` o `híbrida`](/es/guides/server-side-rendering/#enable-on-demand-server-rendering ) activado.
```

Comment:
```suggestion
En Astro, esto se puede hacer con rutas renderizadas bajo demanda en tu proyecto configurado con [salida `servidor` o `híbrida`](/es/guides/server-side-rendering/habilita-la-renderización-de-servidor-bajo-demanda) activado.
```
Hopefully this is the last fix needed!

We got past the imports, but any links to other pages in docs need to also have translated section headers if they exist!  :raised_hands: